recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
-{% set version = "3.0.0" %}
+{% set version = "3.0.1" %}
 
 package:
   name: compliance-checker
   version: {{ version }}
 
 source:
   fn: compliance-checker-{{ version }}.tar.gz
-  url: https://github.com/ioos/compliance-checker/archive/{{ version }}.tar.gz
-  sha256: f49d58d44ba7e75509317dd2b8ec5ee68e5d0e4a90ccab66ae28895e5a7c3276
+  url: https://codeload.github.com/ioos/compliance-checker/tar.gz/{{version}}
```

Comment:
Github redirects from the old URL structure tot his one. I didn't know what libcurl arguments this used so I errored on the side of safety and used the fully resolved URL instead of the redirecting one.

```
curl -i https://github.com/ioos/compliance-checker/archive/3.0.1.tar.gz
HTTP/1.1 302 Found
Server: GitHub.com
Date: Thu, 02 Feb 2017 20:15:27 GMT
Content-Type: text/html; charset=utf-8
Transfer-Encoding: chunked
Status: 302 Found
Cache-Control: no-cache
Vary: X-PJAX
Location: https://codeload.github.com/ioos/compliance-checker/tar.gz/3.0.1
X-UA-Compatible: IE=Edge,chrome=1
Set-Cookie: logged_in=no; domain=.github.com; path=/; expires=Mon, 02 Feb 2037 20:15:27 -0000; secure; HttpOnly
Set-Cookie: _gh_sess=eyJzZXNzaW9uX2lkIjoiNjdiOWVlOWZhNDliODFlMTZkODExMTg3MjZmZTM4MzIiLCJzcHlfcmVwbyI6Imlvb3MvY29tcGxpYW5jZS1jaGVja2VyIiwic3B5X3JlcG9fYXQiOjE0ODYwNjY1Mjd9--cf96d5ac60d61847239d3323cb78729ac76b25d8; path=/; secure; HttpOnly
X-Request-Id: d75530b6e9114d31b3dc2d607d17aec2
X-Runtime: 0.018873
Content-Security-Policy: default-src 'none'; base-uri 'self'; block-all-mixed-content; child-src render.githubusercontent.com; connect-src 'self' uploads.github.com status.github.com collector.githubapp.com api.github.com www.google-analytics.com github-cloud.s3.amazonaws.com wss://live.github.com; font-src assets-cdn.github.com; form-action 'self' github.com gist.github.com; frame-ancestors 'none'; img-src 'self' data: assets-cdn.github.com identicons.github.com collector.githubapp.com github-cloud.s3.amazonaws.com *.githubusercontent.com; media-src 'none'; script-src assets-cdn.github.com; style-src 'unsafe-inline' assets-cdn.github.com
Strict-Transport-Security: max-age=31536000; includeSubdomains; preload
Public-Key-Pins: max-age=5184000; pin-sha256="WoiWRyIOVNa9ihaBciRSC7XHjliYS9VwUGOIud4PB18="; pin-sha256="RRM1dGqnDFsCJXBTHky16vi1obOlCgFFn/yOhI/y+ho="; pin-sha256="k2v657xBsOVe1PQRwOsHsw3bsGT2VzIqz5K+59sNQws="; pin-sha256="K87oWBWM9UZfyddvDfoxL+8lpNyoUB2ptGtn0fv6G2Q="; pin-sha256="IQBnNBEiFuhj+8x6X8XLgh01V9Ic5/V3IRQLNFFc7v4="; pin-sha256="iie1VXtL7HzAMF+/PVPR9xzT80kQxdZeJ+zduCB3uj0="; pin-sha256="LvRiGEjRqfzurezaWuj8Wie2gyHMrW5Q06LspMnox7A="; includeSubDomains
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
Vary: Accept-Encoding
X-Served-By: e5e9a64d3529b20942262801723f444a
X-GitHub-Request-Id: A519:11B38:53829C4:87686FA:5893935E

<html><body>You are being <a href="https://codeload.github.com/ioos/compliance-checker/tar.gz/3.0.1">redirected</a>.</body></html>
```